.github/workflows/linux_sanitizers.yml

```diff
@@ -4,16 +4,35 @@ on:
     # run daily at 00:00
     - cron: '0 0 * * *'
   workflow_dispatch:
+    inputs:
+      # to be defined for specified test scopes (e.g. CPU func. tests)
+      sanitizerMode:
+        description: 'Sanitizer mode for func_tests (All || AddressAndLeak || UndefinedBehavior || Thread).'
+        required: false
+        default: 'All'
+      gtestFilter:
+        description: 'Filter for func_tests.'
+        required: false
+        default: '*smoke*'
+  pull_request:
+    paths:
+    - '.github/workflows/linux_sanitizers.yml'
 
 concurrency:
   # github.ref is not unique in post-commit
-  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-linux-sanitizers
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-linux-sanitizers-tests
```

Comment:
the group affects the both build and test jobs, linux-sanitizers is more generic.